### PR TITLE
Update connextsepolia metadata

### PR DESCRIPTION
**Pull request by Blondelo**
Connext have rebrand in Everclear. 
So I have changed the name by the correct new one and explorer, api as well. Have a good day!

### CODE REVIEW

File: chains/connextsepolia/metadata.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 blockExplorers:
-  - apiUrl: https://connext-sepolia.blockscout.com/api
+  - apiUrl: https://scan.testnet.everclear.org/api
     family: blockscout
-    name: Connext Explorer
-    url: https://connext-sepolia.blockscout.com
+    name: Everclear Explorer
```
Comment (paulbalaji) [inline chains/connextsepolia/metadata.yaml:4]:
```suggestion
    name: Everclear Testnet Explorer
```

File: chains/connextsepolia/metadata.yaml [hunk 2]
```diff
@@ -11,13 +11,13 @@ chainId: 6398
...
   url: https://everclear.org
-displayName: Connext Sepolia
+displayName: Everclear Sepolia
 domainId: 6398
 gasCurrencyCoinGeckoId: ethereum
 index:
   from: 4950
 isTestnet: true
-name: connextsepolia
+name: everclearsepolia
```
Comment (paulbalaji) [inline chains/connextsepolia/metadata.yaml:20]: changing the name is a breaking change. we'll keep it connextsepolia for now
```suggestion
name: connextsepolia
```

**Review by paulbalaji — COMMENTED** [review]
you know the drill, add changeset with `yarn changeset add`

**Comment by Blondelo** [comment]
Okay no problem. The mainnet anme appears in everclear so that's why I wanted to changed it too. I think it's all good now

**Review by jmrossy — APPROVED** [review]

**Pull request merged**